### PR TITLE
Tracking Implants

### PR DESCRIPTION
**Pull request by akenworthy87**
Removes the clunky unintuitive ID number system from tracking implants and replaces it to show the mob's name, as chemical implants already did, and the teleporter already did re: trackers
This is on both the Prisoner Management Console and the hand tracker thingamabob.

Also:
Added a line break on chemical implants on the prisoner management computer so the read out doesn't look like arse. (see here https://www.dropbox.com/s/nawkd06ztqkmkqe/Screenshot%202015-03-28%2015.16.06.png?dl=0)

:cl: Malkevin
tweak: Tracking implants have been upgraded. The old clunky manually adjusted number system has been replaced with an automated ID scanner, this will show the implant carrier's name on both the Prisoner Management Console and the hand tracker. Happy deep striking!
/:cl:

### CODE REVIEW

**Comment by neersighted** [comment]
tgui please...

File: code/game/objects/items/weapons/implants/implant_track.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "Track with this."
 	activated = 0
 	origin_tech = "materials=2;magnets=2;programming=2;biotech=2"
-	var/id = 1
+	var/id = 1	//this should be redundent now but I'm leaving it incase its still used somewhere in the spaghetti we call ss13 code
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/implants/implant_track.dm]: comment it out and compile, if it complains, fix it, if it doesn't complain, remove the commented line altogether.

**Comment by freerealestate** [comment]
:+1:

File: code/game/objects/items/weapons/implants/implant_track.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/objects/items/weapons/implants/implant_track.dm]: Why not remove it and see if any errors pop up?

**Comment by duncathan** [comment]
When did you get unbanned?

**Comment by RemieRichards** [comment]
Also way to do an oldschool changelog ya nerd.

**Comment by Malkevin** [comment]
>When did you get unbanned?
Who knows, I randomly found I could post here again so felt like pushing this thing I did last year.

**Comment by Malkevin** [comment]
>Also way to do an oldschool changelog ya nerd.

https://www.youtube.com/watch?v=Yl1mH0DROQM

**Comment by Malkevin** [comment]
There, I think I've done the change log right...

**Comment by RemieRichards** [comment]
@Malkevin well you have two now.
the old method still works it just, in future, it's waaaaaay easier to do it in the pr body.

**Pull request merged**